.eslintrc.cjs

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-undef */
```

Comment:
why do we have this eslint disabe comment

Reply:
Fixed!